src/Packetery/Module/Checkout/CurrencySwitcherService.php

```diff
@@ -40,7 +61,9 @@ public function getConvertedPrice( float $price ): float {
 		 *
 		 * @since 1.4
 		 */
-		return (float) apply_filters( 'packetery_price', $price );
+		$price = (float) $this->wpAdapter->applyFilters( 'packetery_price', $price );
```

Comment:
proč to nemůže být pořád inline?

Reply:
Vynucujeme řádek před return a dojde pak k odpojení komentáře filtru.

Reply:
Ten máš před komentářem. Ten komentář je součást toho returnu. :) Takže to ti PHPCS pohlídá. Jak bys to pak jinak řešil v případě jedno řádkového komentáře /** @var ... */ ? :)